libpod/oci_conmon_common.go

```diff
@@ -1114,10 +1147,11 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		args = append(args, []string{"--exit-command-arg", arg}...)
 	}
 
-	// Pass down the LISTEN_* environment (see #10443).
 	preserveFDs := ctr.config.PreserveFDs
+
+	// Pass down the LISTEN_* environment (see #10443).
 	if val := os.Getenv("LISTEN_FDS"); val != "" {
-		if ctr.config.PreserveFDs > 0 {
+		if preserveFDs > 0 {
```

Comment:
Right now `LISTEN_FDS` is ignored when `--preserve-fds` is used. 
Should the same happen when `--preserve-fd` is used?

Reply:
yes, let's ignore it also with `--preserve-fd`